style.css

```diff
@@ -650,13 +650,208 @@ header {
 }
```

Comment:
- [ Optional ] Kindly I recommend you to make your mobile version responsive from 375px-768px size screen until Desktop screen size, Your Desktop version start from the screen size 768px to full screen size is responsive so  Kindly implemented for the mobile screen size.
- Please refer this [video](https://drive.google.com/file/d/1pw9ghyyAStVsObOGiWhBr_gN_ttaurWR/view?usp=share_link) for your mobile screen size Project design